.github/workflows/e2e.yml

```diff
@@ -1,25 +1,101 @@
+            - name: Set up Python 3.7
+              uses: actions/setup-python@v1
+              with:
+                  python-version: 3.7
+            - uses: actions/cache@v1
+              name: Cache pip dependencies
+              id: pip-cache
+              with:
+                  path: ~/.cache/pip
+                  key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+                  restore-keys: |
+                      ${{ runner.os }}-pip-
+            - name: Install python dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  python -m pip install $(grep -ivE "psycopg2" requirements.txt) --no-cache-dir --compile
+                  python -m pip install psycopg2-binary --no-cache-dir --compile
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12
+            - name: Get yarn cache directory path
+              id: yarn-dep-cache-dir-path
+              run: echo "::set-output name=dir::$(yarn cache dir)"
+            - uses: actions/cache@v1
+              name: Setup Yarn dep cache
+              id: yarn-dep-cache
+              with:
+                  path: ${{ steps.yarn-dep-cache-dir-path.outputs.dir }}
+                  key: ${{ runner.os }}-yarn-dep-${{ hashFiles('yarn.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-yarn-dep-
+            - name: Yarn install deps
+              run: |
+                  yarn install --frozen-lockfile
+              if: steps.yarn-dep-cache.outputs.cache-hit != 'true'
+            - uses: actions/cache@v1
+              name: Setup Yarn build cache
+              id: yarn-build-cache
+              with:
+                  path: frontend/dist
+                  key: ${{ runner.os }}-yarn-build-${{ hashFiles('frontend/src/') }}
+                  restore-keys: |
+                      ${{ runner.os }}-yarn-build-
+            - name: Yarn build
+              run: |
+                  yarn build
+              if: steps.yarn-build-cache.outputs.cache-hit != 'true'
+            - name: Boot PostHog
+              env:
+                  REDIS_URL: 'redis://localhost'
+                  DATABASE_URL: 'postgres://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres'
+                  DISABLE_SECURE_SSL_REDIRECT: 1
+                  OPT_OUT_CAPTURE: 1
+                  DEBUG: 1
+              run: |
+                  mkdir -p cypress/screenshots 
+                  ./bin/docker-migrate
+                  ./bin/docker-worker &
+                  ./bin/docker-server &
+            - name: Cypress run
+              uses: cypress-io/github-action@v1
+              with:
+                  config-file: cypress.json
+            - name: Archive test screenshots
+              uses: actions/upload-artifact@v1
+              with:
+                  name: screenshots
+                  path: cypress/screenshots
```

Comment:
Where do we access these screenshots?

Reply:
They only are captured if cypress tests fails, but you can grab them at the top right of the log page:
![image](https://user-images.githubusercontent.com/391319/83949302-d6192200-a7d7-11ea-8f8c-a12bd246b7c2.png)